openedx/core/djangoapps/credit/tests/test_models.py

```diff
@@ -83,3 +90,96 @@ def add_credit_course(self):
         credit_course = CreditCourse(course_key=self.course_key, enabled=True)
         credit_course.save()
         return credit_course
+
+
+@ddt.ddt
+class CreditRequestModelTests(TestCase):
+    """
+    Tests for models used to track credit requests and status.
+    """
+
+    USERNAME = u"rön"
+    PROVIDER_ID = u"høgwarts"
+
+    def setUp(self):
+        """Create a course and provider. """
+        super(CreditRequestModelTests, self).setUp()
+        self.course_key = CourseKey.from_string("edX/DemoX/Demo_Course")
+        self.credit_course = CreditCourse.objects.create(course_key=self.course_key)
+        self.credit_provider = CreditProvider.objects.create(provider_id=self.PROVIDER_ID)
+
+    def test_credit_request_current_status(self):
+        request = CreditRequest.objects.create(
```

Comment:
Docstrings would be really helpful for future development and understanding these tests and their expectations.